app/build.gradle

```diff
@@ -535,6 +535,7 @@ dependencies {
     implementation deps.lifecycle.viewmodel
     implementation deps.lifecycle.java8
     implementation deps.lifecycle.process
+    implementation deps.support.core
```

Comment:
Shouldn't we add this with the other `chromiumImplementation` as it looks like we don't need it for gecko?